404.md

```diff
@@ -0,0 +1,7 @@
+---
+permalink: /404.html
+---
+
+# Error 404
+
+How'd you get here! Anyways, you seem to be lost.
```

Comment:
Missing memes.
Also, improper grammar. How'd you get here is a question, not an exclamation.

Fix these very important errors.



* also we need a cat picture *